### PR TITLE
Chore: Add migrations needed for multilayered map entities

### PR DESCRIPTION
**Pull request by Lennox2536**
Entities proposal for the multilayered map.

Diagram:
image

### CODE REVIEW

**Comment by simon-the-shark** [comment]
1. We prolly dont need to connect this to buildings. I see why you did that, but I don't thing we're gonna use it. But if you will leave it, please make it optional

2. Aeds need place for Photo

**Comment by simon-the-shark** [comment]
3. We prolly want adresses for every type. Not sure what we're gonna place there, but it's a good thing to have

**Comment by czaja307** [comment]
The code looks solid overall. Just make the adjustments suggested by Simon, and it should be good to go.

File: app/enums/building_icon.ts
Comment (czaja307) [inline app/enums/building_icon.ts:1]: we'll have to consult what other icon types mobile want, but it's good for now

**Review by czaja307 — APPROVED** [review]
LGTM

**Comment by simon-the-shark** [comment]
> Entities proposal for the multilayered map.
> 
> Diagram: image

@czaja307 is this most up to date ERP? Cause @odominiak wants to make some "scraping"???

**Comment by czaja307** [comment]
yup, it's up to date

**Pull request merged**